### PR TITLE
fix: hypersonic preferences command

### PR DESCRIPTION
**Pull request by prrrcl**
## Description

Fix the extension command preferences 

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Looks good to me, only have a single question that might be better for the UX 😄 

Let me know what you think

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/hypersonic/src/components/empty-list.tsx [hunk 1]
```diff
@@ -106,6 +107,12 @@ export function EmptyList({ notionDbUrl, getInitialData }: EmptyListProps) {
         <ActionPanel>
           <WhatHaveIDoneAction />
           <ReauthorizeAction />
+          <Action
+            title="Open Extension Preferences"
+            icon={Icon.Gear}
+            onAction={openCommandPreferences}
+            shortcut={{ modifiers: ['opt'], key: 'u' }}
```
[the remark this answers is not in the document]
Comment (prrrcl) [reply extensions/hypersonic/src/components/empty-list.tsx]: why not? 🤗

File: extensions/hypersonic/src/views/todo-list.tsx [hunk 2]
```diff
@@ -129,7 +129,7 @@ export function ToDoList() {
                   <Action
                     title="Open Extension Preferences"
                     icon={Icon.Gear}
-                    onAction={openExtensionPreferences}
+                    onAction={openCommandPreferences}
                     shortcut={{ modifiers: ['opt'], key: 'u' }}
```
[the remark this answers is not in the document]
Comment (prrrcl) [reply extensions/hypersonic/src/views/todo-list.tsx]: why not? 🤗

**Review by pernielsentikaer — APPROVED** [review]
Let's get it merged 🔥 Thanks @prrrcl 🔥

**Pull request merged**